### ISSUE TITLE
Rename to ember-inspector-study

### ISSUE THREAD

**Issue by payne-chris-r** [ga-wdi-boston/ember-inspector#13]
I think this entire talk can be made into a study instead of a talk. It would need to be reformatted slightly, and maybe put a little more emphasis on `ember-data`, but I think it's possible. The ember inspector is something they should be familiar with and that can help them, but I don't think we need to dedicate a unit of class-time to it. Thoughts?

**Comment by gaand** [comment]
Does it use this https://guides.emberjs.com/v2.11.0/ember-inspector/ ❓

**Comment by jrhorn424** [comment]
I agree. We should use it liberally during talks but not teach it specifically.

Similar to how we use, but don't teach the web inspector specifically.

**Comment by MicFin** [comment]
This should be renamed to `ember-inspector-study`.  Some files from the `ember-template` where we would generally have `ember-inspector-study` have the name `super-rentals` from the exercise.

**Comment by BenGitsCode** [comment]
What do you mean by that @MicFin The repo name should be changed to `ember-inspector-study` but what do you mean, some of the files like the package.json title or something like that?

**Comment by MicFin** [comment]
Yes, I think the list of files that require name changes are below.  This study does not seem to follow our current pattern due to `super-rentals` app name being used where generally `ga-wdi-boston.ember-inspector-study` would be.  That may be fine, but it is not clear that it should be that way.

>Replace all instances of 'ga-wdi-boston.ember-template' with your app name. This includes package.json, bower.json, app/index.html, tests/index.html, and config/environment.js, possibly others.
https://github.com/ga-wdi-boston/ember-template#installation

**Comment by payne-chris-r** [comment]
This change requires a good deal of knowledge of `ember` and our `ember` sequence as a whole. Please loop me in when this gets prepped in the future!
